tests/ops/service/privacy_request/test_bigquery_enterprise_privacy_request.py

```diff
@@ -28,29 +28,38 @@
 
 PRIVACY_REQUEST_TASK_TIMEOUT = 5
 # External services take much longer to return
-PRIVACY_REQUEST_TASK_TIMEOUT_EXTERNAL = 100
+PRIVACY_REQUEST_TASK_TIMEOUT_EXTERNAL = 150
 
 
 @pytest.mark.integration_bigquery
 @pytest.mark.integration_external
 @pytest.mark.parametrize(
     "dsr_version",
-    ["use_dsr_3_0", "use_dsr_2_0"],
+    ["use_dsr_2_0", "use_dsr_3_0"],
+)
+@pytest.mark.parametrize(
+    "bigquery_fixtures",
+    [
+        "bigquery_enterprise_test_dataset_config",
+    ],
```

Comment:
Should the fixture with partitioning meta be listed here?
```suggestion
@pytest.mark.parametrize(
    "bigquery_fixtures",
    [
        "bigquery_enterprise_test_dataset_config",
        "bigquery_enterprise_test_dataset_config_with_partitioning_meta",
    ],
)
```

Reply:
We actually test the partitioning access within the `test_erasure_request` test, since the first step is to create test data, then access it. We assert there that we have expected data using partitioning. 

The reasoning for this is since each query / test against the enterprise BQ dataset is resource-intensive, we want to reduce duplicate test cases.